Update formal-review.md (#506)
mattermost/mattermost-handbook: operations/workplace/people/performance-reviews-50/formal-review-process.md

# Review Process for People Managers

This is the review process for the Mid-Year and Winter (formal) Performance Reviews. Note: Links to the peer request and peer feedback forms will be either included or specified in each review timeline. Also, calibration will differ between Mid-Year and Winter. See Timeline for detailed instructions.

## Confirm level expectations for roles are up to date

> If any level descriptions need to be updated or written, Managers complete it before the review cycle begins.

Reviewer: ```suggestion
If any level descriptions need to be updated or written, Managers complete them before the review cycle begins.
```

## Manager training

> Before each review cycle, managers read the performance review chapter in High Output Management. Managers will be invited to attend [Manager Training on Performance Management at Mattermost](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#manager-training) at the beginning of the [Annual Performance Review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#what-is-the-process). If you don't have a copy, use [this form](https://forms.gle/SUWWvZZGqg5B4d1aA) to request a copy.

Reviewer: ```suggestion
Before each review cycle, Managers read the performance review chapter in High Output Management. Managers will be invited to attend [Manager Training on Performance Management at Mattermost](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#manager-training) at the beginning of the [Annual Performance Review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#what-is-the-process). If you don't have a copy, use [this form](https://forms.gle/SUWWvZZGqg5B4d1aA) to request a copy.
```

## Feedback panel requests

> Managers will provide peers for each direct report who can most effectively provide feedback on achievements and performance over the past year \(okay to include community members in addition to staff\).

Reviewer: ```suggestion
Managers will provide peers for each direct report who can most effectively provide feedback on achievements and performance over the past year \(this can include community members in addition to staff\).
```

## Peer feedback

> Your selected peers will receive a request to complete the [annual review feedback survey](https://docs.google.com/forms/d/1JNlDCAOgUjjLSJj5EoNe7Pl5kqN6KN7LaEIlM5L9LXw/edit) within two weeks of receipt. On average, teammates should be completing feedback surveys for 3 to 6 of their peers.

Reviewer: ```suggestion
Your selected peers will receive a request to complete the [annual review feedback survey](https://docs.google.com/forms/d/1JNlDCAOgUjjLSJj5EoNe7Pl5kqN6KN7LaEIlM5L9LXw/edit) within two weeks of receipt. On average, teammates should be completing feedback surveys for 3 - 6 of their peers.
```

## Self reflection

Each staff member will receive a BambooHR notification via email to complete their Self-Assessment in BambooHR.

## Draft of annual review

Once a staff member completes their Self-Assessment in BambooHR, Managers will be notified. The Self-Assessment and Manager Review Forms are combined in one document in BambooHR. Managers will read all feedback from the surveys and incorporate their feedback based on their own performance expectations of this role. Managers can draft performance reviews.

**Note:** Once a Manager completes the review via their signature in BambooHR, the feedback form will be automatically shared with the direct report. It is helpful to share the review ahead of the 1:1 meeting to give an opportunity to read feedback, but please plan timing accordingly.

## Calibration

> All people managers within the same [department](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#departments) or other group that the Team [MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence) indicates.

Reviewer: ```suggestion
All people Managers within the same [department](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#departments) or other group that the Team [MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence) indicates.
```

> Managers will share their review 'expectation' ratings (meets, exceeds, does not meet), any recommendations for a [level](https://docs.google.com/document/d/1XNGYOHouoY42YYmFHNrhu-vHDtWVrPL5E8M_BpwF9iU/edit) change and/or a promotion, asynchronously using a confidential google sheet that is shared with the calibration group. During the meeting, managers share feedback on any performance metric and apply [high standards](https://handbook.mattermost.com/company/about-mattermost#leadership-principles) in evaluation against the criteria for someone's role and level.

Reviewer: ```suggestion
Managers will share their review 'expectation' ratings (meets, exceeds, does not meet), any recommendations for a [level](https://docs.google.com/document/d/1XNGYOHouoY42YYmFHNrhu-vHDtWVrPL5E8M_BpwF9iU/edit) change and/or a promotion, asynchronously using a confidential Google Sheet that's shared with the calibration group. During the meeting, Managers share feedback on any performance metric and apply [high standards](https://handbook.mattermost.com/company/about-mattermost#leadership-principles) in evaluation against the criteria for staff members' role and level.
```

## Compensation review and change memos

All compensation changes require approval of the VP of Finance & Operations and the CEO. CTO approval is required ahead of CEO approval for all Engineering compensation changes.

> 1. HR will review compensation data with each Team MLT 1x1, asynchronously.

Reviewer: ```suggestion
1. HR will review compensation data with each Team MLT 1:1, asynchronously.
```

> 2. HR, Team MLT + required comp approvers meet to discuss live with email approval to follow and finalize with all attendees CC'd.

Reviewer: ```suggestion
2. HR, Team MLT, and required comp approvers meet to discuss, with email approval to follow and finalize with all attendees CC'd.
```
